qiskit/circuit/measure.py

```diff
@@ -38,6 +34,7 @@ def reapply(self, circuit):
         self._modifiers(circuit.measure(self.qargs[0], self.cargs[0]))
 
 
+@_2q_gate
```

Comment:
One place where this could be confusing:
```
qr = QuantumRegister(3)
cr = QuantumRegister(1)
qc = QuantumCircuit(qr, qc)

qc.measure(qr, cr)
```
would consecutively measure several qubits onto a single cbit, overwriting every measure but the last. Might be worth a warning in this case, or exempting `measure` from `1->N` broadcasting.

Reply:
The recent commit addresses this with your parameterized decorator suggestion. Thanks.